style.css

```diff
@@ -1,825 +1,810 @@
 }
 
-.text__title {
-  width: 549px;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 80px;
-  line-height: 82px;
-  margin-bottom: 0;
+.second__block-color-gray,
+.second__block-color-pink,
+.second__block-color-blue,
+.second__block-color-green {
+	width: 23px;
+	height: 23px;
+	border-radius: 50%;
 }
 
-.text__subtitle {
-  font-family: Montserrat, sans-serif;
-  font-style: normal;
-  font-weight: normal;
-  font-size: 15px;
-  line-height: 21px;
-  letter-spacing: .5px;
-  margin-bottom: 52px;
+.second__block-color-gray {
+	background-color: #dedede;
 }
 
-.collection__next-text {
-  text-decoration: none;
-  color: black;
-  font-style: normal;
-  font-size: 13px;
-  border: 1px solid black;
-  padding: 10px 23px;
-  margin-right: 52px;
+.second__block-color-pink {
+	background-color: #f29f9f;
+	margin-left: 9.14px;
 }
 
-.collection__other-text {
-  display: inline-block;
-  text-decoration: none;
-  color: black;
+.second__block-color-blue {
+	background-color: #a8bff9;
+  margin-left: 9.14px;
+}
+
+.second__block-color-green {
+  background-color: #90bd8f;
+  margin-left: 9.14px;
+}
+
+.second__block-coust {
+	font-size: 20px;
+	font-weight: 900;
+	margin: 0;
+}
+
+.second__block {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin-bottom: 42px;
+}
+
+.second__add {
+  font-weight: 500;
   font-size: 13px;
-  letter-spacing: 0;
-  margin-right: 14px;
+  text-decoration: none;
+  color: #000;
+  padding: 10px 48px;
+  border: 1px solid #000;
+  margin-left: 31px;
 }
 
-.collection__scroll {
-  position: absolute;
-  width: 27px;
-  height: 46px;
-  left: 49%;
-  top: 815px;
-  border: 2px solid #000;
-  box-sizing: border-box;
-  border-radius: 15px;
+.third {
+  min-width: 320px;
+  height: 597px;
+  background-color: #ebe3d8;
+  position: relative;
 }
 
-.collection__scroll-elem {
-  position: absolute;
-  width: 2.45px;
-  height: 8.21px;
-  left: 50%;
-  top: 826.5px;
+.third__line {
+  width: 41px;
+  height: 597px;
   background: #000;
-  border-radius: 19px;
+  margin-left: 100px;
 }
 
-.collection__next-text:hover,
-.collection__other-text:hover {
-  opacity: .5;
-  transition: 1s ease-out .2s;
+.third__img {
+	position: absolute;
+  min-width: 320px;
+  right: 12px;
+  left: 12px;
+  top: 34px;
 }
 
-.vectormob {
-  display: none;
+.third__img>div {
+  background: url(img/third-img.png) no-repeat;
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  height: 262px;
 }
 
-@media (max-width: 1235px) {
-  .first {
-    max-width: 1235px;
-    width: 100%;
-    display: grid;
-    grid-template-columns: 35.2% 64.8%;
-    grid-template-rows: 719px;
-  }
+.third__block {
+  width: 305px;
+  height: 195px;
+  background: #fae8e4;
+  top: 274px;
+  position: absolute;
+  margin: 0 auto;
+  left: 0;
+  right: 0;
+}
 
-  .header__logo {
-    font-size: 24px;
-    letter-spacing: 2px;
-    margin: 5px 41px;
-  }
+.third__container {
+	width: 256px;
+	height: 115px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)